app/models/setting/remote_execution.rb

```diff
@@ -28,7 +28,10 @@ def self.load_defaults
                  { :collection => Proc.new {Hash[SSHExecutionProvider::EFFECTIVE_USER_METHODS.map{|method| [method, method]}]} }),
         self.set('remote_execution_sync_templates',
                  N_('Whether we should sync templates from disk when running db:seed.'),
-                 true)
+                 true),
+        self.set('remote_execution_ssh_port',
+                N_('Port to use for SSH communication. Default port 22.'),
```

Comment:
Could you make a not about ability to override this in host param in the description, similarly as it's described in remote_execution_ssh_user https://github.com/dhlavac/foreman_remote_execution/blob/4418fbcc758442eb58111297d842060d013a5285/app/models/setting/remote_execution.rb#L18 ?